nevergrad/functions/cycling/WomensTeamPursuit.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+from .TeamPursuit import TeamPursuit
+from .Cyclist import Cyclist
+from .SimulationResult import SimulationResult
+import sys
+import math
+
+class WomensTeamPursuit(TeamPursuit):
```

Comment:
how much of this is shared with MensTeamPursuit? There seems to be a lot of similar code. Factorizing it will be helpful on the long term, at the very least for maintenance (and some remarks left on the other class are similar for this class, so instead of duplicating work, it's actually simpler to find a way to merge them ;) )